ocrd_utils/ocrd_utils/config.py

```diff
@@ -151,6 +151,11 @@ def _ocrd_download_timeout_parser(val):
         description="Default address of Workspace Server to connect to (for `ocrd network client workspace`).",
         default=(True, ''))
 
+config.add("OCRD_NETWORK_WORKER_QUEUE_CONNECT_ATTEMPTS",
+    description="Number of attempts for a worker to create its queue. Helpfull if the rabbitmq-server needs time to be fully started",
+    parser=int,
+    default=(True, 1))
```

Comment:
The default should be 3?

Reply:
I thought not to wait by default only when "needed" and that could be configured via ENV

Reply:
Makes sense, I committed accordingly.